### PR TITLE
[Forwardport] Corrected return message from ProductRuleTest.php

### PR DESCRIPTION
**Pull request by sanganinamrata**
### Original Pull Request 
 https://github.com/magento/magento2/pull/16721

### Description

Corrected return message from ProductRuleTest.php

### Fixed Issues (if relevant)

N/A

### Manual testing scenarios

N/A

### Contribution checklist
 - [X] Pull request has a meaningful description of its purpose
 - [X] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds on Travis CI are green)

### CODE REVIEW

**Comment by magento-engcom-team** [comment]
Hi @sanganinamrata. Thank you for your contribution 
Here is some useful tips how you can test your changes using Magento test environment.
Add the comment under your pull request to deploy test or vanilla Magento instance:
- `@magento-engcom-team give me test instance` - deploy test instance based on PR changes
- `@magento-engcom-team give me {$VERSION} instance` - deploy vanilla Magento instance

For more details, please, review the [Magento Contributor Assistant](https://github.com/magento/magento2/wiki/Magento-Contributor-Assistant) documentation

File: dev/tests/integration/testsuite/Magento/CatalogRule/Model/Indexer/ProductRuleTest.php [hunk 1]
```diff
@@ -67,7 +67,7 @@ public function testReindexWithProductNotVisibleIndividually()
         $this->assertEquals(
             7.5,
             $this->resourceRule->getRulePrice(new \DateTime(), 1, 1, $product->getId()),
-            "Catalog price rule doesn't apply to to product with visibility value \"Not Visibility Individually\""
+            "Catalog price rule doesn't ap product with visibility value \"Not Visibility Individually\""
```
Comment (ihor-sviziev) [inline dev/tests/integration/testsuite/Magento/CatalogRule/Model/Indexer/ProductRuleTest.php]: It looks like mistake. This file wasn't changed in original PR

**Review by ihor-sviziev — CHANGES_REQUESTED** [review]

File: dev/tests/integration/testsuite/Magento/CatalogRule/Model/Indexer/ProductRuleTest.php (hunk 1 above)
Comment (gelanivishal) [reply dev/tests/integration/testsuite/Magento/CatalogRule/Model/Indexer/ProductRuleTest.php]: Done.

**Review by ihor-sviziev — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @ihor-sviziev, thank you for the review.
[ENGCOM-2592](https://jira.corp.magento.com/browse/ENGCOM-2592) has been created to process this Pull Request

**Comment by magento-engcom-team** [comment]
Hi @sanganinamrata. Thank you for your contribution. 
We will aim to release these changes as part of 2.3.0. 
Please check the release notes for final confirmation.

**Pull request merged**